src/content/docs/en/community-resources/talks.mdx

```diff
@@ -30,6 +30,7 @@ Subscribe to [the Astro Community Events Google Calendar](https://calendar.googl
 - [View Transitions: Fact vs. Fiction](https://www.youtube.com/watch?v=iT-3amHK7tA&list=PLqGQbXn_GDmkOsHI7-Wrbv1GgAA4tJZhg&index=29)  by Fred Schott at ViteConf 2023
 - [Bringing React into your content sites with Astro](https://www.youtube.com/watch?v=Fouws8iKPKQ) by Matthew Phillips at Future Frontend 2023
 - [All Things Astro](https://www.youtube.com/watch?v=WKXHFIulZ9g) by Elian Van Cutsem, React Day Berlin 2023
+- [You Don’t Know How to SSR](https://portal.gitnation.org/contents/you-dont-know-how-to-ssr) by Emanuele Stoppa, DevOps.js Conf 2024
```

Comment:
Though rendered the same in the docs page, other places in the file are using the apostrophe `'` instead of right quotation mark `’`

Reply:
Fun fact: The docs site will publish both of these as the right quotation mark! But I think typing the apostophe is easier on some keyboards. (I don't even know how I'd produce the right angled mark.)